pydis_site/static/css/base/base.css

```diff
@@ -0,0 +1,40 @@
+html {
+    overflow:auto;
```

Comment:
this uses 4 spaces and no space between colon and text then the ones below use two spaces and space between colon and text, we should stay consistent